helm/cloud-provider-azure/templates/cloud-provider-azure.yaml

```diff
@@ -17,13 +17,13 @@
 {{- else if eq .Capabilities.KubeVersion.Minor "29" -}}
   {{- $_ := set . "cloudProviderAzureVersion" "v1.29.15" }}
 {{- else if eq .Capabilities.KubeVersion.Minor "30" -}}
-  {{- $_ := set . "cloudProviderAzureVersion" "v1.30.12" }}
+  {{- $_ := set . "cloudProviderAzureVersion" "v1.30.13" }}
```

Comment:
Sorry, just getting to this now. I see new patch releases for cloud-provider-azure are available now since this was first opened, v1.30.14 etc. Is it important that the chart doesn't skip any versions or could we go straight to those latest versions?

Reply:
I don't think there's a need to release a Chart for the now-old patch versions. I'll update this PR, thanks for noticing that!